specification/2.0/README.md

```diff
@@ -956,6 +956,8 @@ The following example shows an image pointing to an external PNG image file and
     ]
 }
 ```
+> **Implementation Note:** When image data is provided by `uri` and `mimeType` is defined, clients should prefer JSON-defined MIME Type over one provided by transport layer.
```

Comment:
> Is this easy to implement?

I don't see any problem with implementing that.

> Do we consider this to be valid glTF? 

Currently, yes. Otherwise, we should change the schema.

>  I would have expected `mimeType` to be ignored if both `uri` and `mimeType` are specified.

There could be a rare case, when `uri` doesn't contain any file extension and server doesn't give meaningful `Content-Type` header value. So JSON-provided `mimeType` could be used instead of guessing image format from magic numbers.